restapi.go

```diff
@@ -1832,6 +1832,129 @@ func (s *Session) ChannelNewsFollow(channelID, targetID string) (st *ChannelFoll
 	return
 }
 
+// ------------------------------------------------------------------------------------------------
+// Functions specific to Direct Messages
+// ------------------------------------------------------------------------------------------------
+
+// UserMessageSend sends a message to a user.
+// userID : The ID of a User.
+// content   : The message to send.
+func (s *Session) UserMessageSend(userID, content string) (*Message, error) {
+	userChannel, err := s.UserChannelCreate(userID)
+	if err != nil {
+		return nil, err
+	}
+
+	return s.ChannelMessageSendComplex(userChannel.ID, &MessageSend{
+		Content: content,
+	})
```

Comment:
tell me if i'm wrong, but it might be better to use this instead
```suggestion
	return s.ChannelMessageSend(userChannel.ID, content)
```

Reply:
The ChannelMessageSend calls ChannelMessageSendComplex the exact same way I do, internally, there is no need to use it because it's there for convenience. It's calling another function for pretty much no reason.